manifests/kadalu-operator.yaml

```diff
@@ -192,7 +192,7 @@ spec:
             capabilities: {}
             privileged: true
           image: docker.io/kadalu/kadalu-operator:latest
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
This is not required, i think we need to remove this

Reply:
Done